app/platform/forms.py

```diff
@@ -46,3 +49,40 @@ class FormForm(FlaskForm):
         widget=GovTextInput(),
     )
     submit = SubmitField(widget=GovSubmitInput())
+
+
+class QuestionTypeForm(FlaskForm):
+    question_data_type = RadioField(
+        "Question type",
+        choices=[
+            (QuestionDataType.TEXT_SINGLE_LINE.name, QuestionDataType.TEXT_SINGLE_LINE.value),
+            (QuestionDataType.INTEGER.name, QuestionDataType.INTEGER.value),
+            (QuestionDataType.TEXT_MULTI_LINE.name, QuestionDataType.TEXT_MULTI_LINE.value),
+        ],
+        validators=[DataRequired("Select a question type")],
+        widget=GovRadioInput(),
+    )
+    submit = SubmitField(widget=GovSubmitInput())
+
+
+class QuestionForm(FlaskForm):
+    text = StringField(
+        "What is the question?",
+        validators=[DataRequired("Enter the question text")],
+        filters=[strip_string_if_not_empty],
+        widget=GovTextInput(),
+    )
+    hint = StringField(
+        "Question hint",
+        filters=[strip_string_if_not_empty],
+        widget=GovTextArea(),
+        description="The question hint will be shown to users when they are answering the question.",
+    )
+    name = StringField(
+        "Question name",
+        validators=[DataRequired("Enter the question name")],
+        description="The question name will be shown when exporting submissions from your users.",
+        filters=[strip_string_if_not_empty],
+        widget=GovTextInput(),
+    )
```

Comment:
<img width="790" alt="image" src="https://github.com/user-attachments/assets/e53a1694-c3e9-4c9c-bb97-6a1e2002cbdb" />

A similar issue here with using the question label for the error summary - I don't think this will scale.

The field label defined here should be the question, so I don't think that should change. But we need some way to build the correct error message; I think our formulaic approach is not working well.

Reply:
Fixed this elsewhere but just missed this one - done now